pkgs/build-support/docker/default.nix

```diff
@@ -820,10 +870,9 @@ rec {
         # Make the image read-only.
         chmod -R a-w image
 
-        echo "Cooking the image..."
-        tar -C image --hard-dereference --sort=name --mtime="@$SOURCE_DATE_EPOCH" --owner=0 --group=0 --xform s:'^./':: -c . | pigz -nT > $out
-
         echo "Finished."
+        echo "To cook and load:"
+        printf "\e[32m%s\e[0m" 'tar -C result/image --dereference --hard-dereference --sort=name --mtime="1" --owner=0 --group=0 --mode=a-w --xform s:'^./':: -c . | docker load'
```

Comment:
More of an idea but this should also work nicely:
```sh
skopeo copy dir:./result/image docker-daemon://image-name:tag-name
```
Skopeo requires to pass the image-name and tag-name to the destination so those could be extracted from the result as well.

Reply:
I ran into some problems using skopeo for this purpose related to it expecting Image Manifest V 2, Schema 2. We're using an older schema. But that is/was the goal.